stack2nix/Main.hs

```diff
@@ -9,6 +9,7 @@ args = Args
        <$> optional (strOption $ long "revision" <> help "revision to use when fetching from VCS")
        <*> optional (strOption $ short 'o' <> help "output file for generated nix expression")
        <*> strArgument (metavar "URI")
+       <*> (flag False True $ long "system-ghc" <> help "pass --system-ghc to stack invocations")
```

Comment:
Please reorder this so the `URI` comes last. It makes `stack2nix --help` more readable.

Otherwise, looks great! Thanks 👍